core/engine/src/value/conversions/serde_json.rs

```diff
@@ -140,6 +140,25 @@ impl JsValue {
                     Ok(Value::Array(arr))
                 } else {
                     let mut map = Map::new();
+                    let mut value_by_prop_key = |property_key| match obj
+                        .borrow()
+                        .properties()
+                        .get(&property_key)
+                        .and_then(|x| x.value().cloned())
+                    {
+                        Some(val) => val.to_json(context),
+                        None => Ok(Value::Null),
+                    };
```

Comment:
We could avoid the `clone` here with extra `map`:
```rust
     .and_then(|x| x.value().map(|v| v.to_json(context)))
{
    Some(val) => val?,
    None => Value::Null,
};
```

Reply:
In the worst case it's still just cloning of a smart pointer so I don't think that it has much overhead costs, but ok, done